docs/EntityProjectile.md

```diff
@@ -51,12 +51,12 @@ ___
 ___
 ### Clear·Projectile·Flags () {: aria-label='Functions' }
 [ ](#){: .rep .tooltip .badge }
-#### void ClearProjectileFlags ( int Flags ) {: .copyable aria-label='Functions' }
+#### void ClearProjectileFlags ( [`ProjectileFlags`](enums/ProjectileFlags.md) Flags ) {: .copyable aria-label='Functions' }
```

Comment:
Don't use ` here please. Function definitions need to have a uniform look. ` Will create a code block in a function definition, which looks odd